src/app/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { NavbarComponent } from '../navbar/navbar.component';
+
+@Component({
+  selector: 'dashboard',
+  templateUrl: './dashboard.component.html',
+  directives: [ NavbarComponent ]
+})
+export class DashboardComponent {
+
+    private isProject: boolean;
```

Comment:
Maybe rename to `haveProjects`? I'm guessing this is to track whether or not there are any projects to list. Or better yet, define the list of `projects` as an array for which we can check the length.